src/mann_whitney.jl

```diff
@@ -145,21 +147,17 @@ function pvalue(x::ExactMannWhitneyUTest; tail=:both)
             end
         elseif tail == :left
             pwilcox(x.U, x.nx, x.ny, true)
-        elseif tail == :right
+        else # tail == :right
             pwilcox(x.U - 1, x.nx, x.ny, false)
-        else
-            throw(ArgumentError("tail=$(tail) is invalid"))
         end
     else
         # Compute exact p-value by enumerating possible ranks in the tied data
         if tail == :both
             min(1, 2 * minimum(mwuenumerate(x)))
```

Comment:
Need to fix this one too.

Reply:
It seems due to the definition of `mwuenumerate` and `x`, `mwuenumerate(x)` is always a `Float64`. So we can use `1.0`.